### DISCUSSION TITLE
Dynamically setting data set for select inputs #5928

### DISCUSSION THREAD

**Discussion by E2dwhy** [formio/ng2-formio#1113] [General]
We are trying to dynamicaly set the data from the Select input

We are trying to dynamicaly set the data from the Select input, that is, listening to key and key down events and update the data of the select dropdown so that we optimize the app to not load the entire dataset that is mainly about 8000 records.
Is there a way to make this work? and if yes can you guide us through?

**Comment by lane-formio** [comment]
https://github.com/formio/formio.js/issues/5928
